dotnetv3/EC2/README.md

```diff
@@ -13,19 +13,41 @@ Amazon EC2 is a web service that provides resizable computing capacity—literal
 
 ## Code examples
 
+### Getting started
+* [Hello Amazon EC2](actions/HelloEc2.cs)
+
 ### Single actions
 Code excerpts that show you how to call individual service functions.
-* [Create a security group](CreateSecurityGroupExample/CreateSecurityGroupExample/CreateSecurityGroup.cs) (`CreateSecurityGroupAsync`)
-* [Create a security key pair](CreateKeyPairExample/CreateKeyPairExample/CreateKeyPair.cs) (`CreateKeyPairAsync`)
-* [Create a VPC](VirtualPrivateCloudExamples/CreateVPCExample/CreateVPC.cs) (`CreateVpcAsync`)
+* [AllocateAddress](actions/EC2Wrapper.cs) (`AllocateAddressAsync`)
```

Comment:
These three should be long form names, not action names, right?

Reply:
Edited according to descriptions in the code examples library.